features/testbot/bearing_param.feature

```diff
@@ -75,7 +75,10 @@ Feature: Bearing parameter
             #| 0    | b  | 189 189  | da          | 180                                   |
             | 0    | 1  | 90 270   | ab,bc,cd,cd    | 0->90,90->0,0->270,270->0           |
             | 1    | d  | 10 10    | bc,bc          | 0->0,0->0                           |
-            | 1    | d  | 90 90    | ab,bc,cd,da,da | 0->90,90->0,0->270,270->180,180->0  |
+            # The target point `a` can belong to either `ab` or `ha` edge.
+            # The latter case leads to "Impossible route between points",
+            # because `ab` and `ha` are in different components
+            #| 1    | d  | 90 90    | ab,bc,cd,da,da | 0->90,90->0,0->270,270->180,180->0 |
```

Comment:
I'm not sure I understand this. We are not trying to snap to `a`, we are trying to snap to a coordinate close to `1` (which is just a coordinate not linked to an OSM node). It should be something in the middle of `ab` (which has bearing 90°).

The target segment for `d` should be unique as well. It is only contained by one segment (`cd`) with 90°. I'm quite confused why the original test specifies `da` as target segment which has bearing `180` and should not be chosen as segment for `d`.